### PR TITLE
Fix starting device plugin issue

### LINKED ISSUE #71: Use "sh" instead of "bash" in pod-sriovdp.yaml template
Since the image that is being used is "alpine" this `command: [ "/bin/bash", "-c", "--" ]` should be `command: [ "/bin/sh", "-c", "--" ]`, also please consider updating the README.md with this change.

### PR DESCRIPTION
**Pull request by Mmduh-483**
Due to change to alpine image and since "bash" command not
existing it shoud be changed to "sh" command

Fixes #71

### CODE REVIEW

**Comment by zshi-redhat** [comment]
Thanks for the fix!  This template is so outdated.
Would you mind also fix other issues in the ``pod-sriovdp.yaml`` template?
for example:
1) the ``/etc/pcidp/config.json`` config file is not mounted to container
2) ``/sys/class/net`` is not used as mountPath to discover network devices, it should be ``/sys`` now.
3) ``pod-sriovdp.yaml`` doesn't execute ``sriovdp`` binary by default, perhaps adding a real command and args.

**Comment by Mmduh-483** [comment]
@zshi-redhat 
Thanks for you comment
1- Regarding `/etc/pcidp/config.json` I did that in PR #74 
2- For the `net` volume is changed from `/sys/class/net` to `/sys` and it is confirmed to be working fine, right?
3- sriovdp binary will start working by default when create a the pod now, is there any default arguments you want to add  `/bin/sriovdp --key1 val1 --key2 val2`? or just running the binary with its default arguments `./bin/sriovdp`?

**Comment by zshi-redhat** [comment]
> @zshi-redhat
> Thanks for you comment
> 1- Regarding `/etc/pcidp/config.json` I did that in PR #74
> 2- For the `net` volume is changed from `/sys/class/net` to `/sys` and it is confirmed to be working fine, right?
Yes, it should work with ``/sys`` now as we are not using ``/sys/class/net`` but ``/sys/bus/pci/device``, see [here](https://github.com/intel/sriov-network-device-plugin/blob/master/pkg/utils/utils.go#L29). If it doesn't work, then there is a bug.

> 3- sriovdp binary will start working by default when create a the pod now, is there any default arguments you want to add `/bin/sriovdp --key1 val1 --key2 val2`? or just running the binary with its default arguments `./bin/sriovdp`?
Please take a look at the [sriovdp-daemonset](https://github.com/intel/sriov-network-device-plugin/blob/master/images/sriovdp-daemonset.yaml) template which runs sriov device plugin as a DaemonSet, it has all the necessary args and mount volumes set, and it's the recommended way to run sriov device plugin. This pod-sriovdp I think is more for development testing purpose.

**Comment by Mmduh-483** [comment]
@zshi-redhat I added the changes you asked except for mount `/etc/pcidp` since it is in  PR  #74

File: deployments/pod-sriovdp.yaml [hunk 1]
```diff
@@ -7,13 +7,13 @@ spec:
   - name: sriov-device-plugin 
     image: nfvpe/sriov-device-plugin:latest
     imagePullPolicy: IfNotPresent
-    command: [ "/bin/bash", "-c", "--" ]
-    args: [ "while true; do sleep 300000; done;" ]
+    command: [ "/bin/sh", "-c", "--" ]
+    args: [ "sriovdp --logtostderr 10; while true; do sleep 300000; done;" ]
```
Comment (zshi-redhat) [inline deployments/pod-sriovdp.yaml]: I think sriovdp will run Infinitely until it receives the ``sigCh`` signal, so perhaps remove the ``while`` loop in the command args?

**Comment by ahalim-intel** [comment]
@Mmduh-483 I tried to resolve conflicts after merging #74 . Can you resolve this locally and update this PR. Better maybe create new clean PR after a rebase and close this one.

**Comment by Mmduh-483** [comment]
@ahalim-intel , due to some issue with my repository I will do this in another pull request #81

**Pull request closed without merging**